stagecoach/jobs/publishers/publish-all/app.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import json
+import os
+
+import boto3
+from empiric.core import SpotEntry
+from empiric.core.logger import get_stream_logger
+from empiric.publisher.assets import EMPIRIC_ALL_ASSETS
+from empiric.publisher.client import EmpiricPublisherClient
+from empiric.publisher.fetchers import (
+    BitstampFetcher,
+    CexFetcher,
+    CoinbaseFetcher,
+    FtxFetcher,
+    GeminiFetcher,
+    TheGraphFetcher,
+)
+
+logger = get_stream_logger()
+
+
+def handler(event, context):
+    entries_ = asyncio.run(_handler(EMPIRIC_ALL_ASSETS))
+    serialized_entries_ = SpotEntry.serialize_entries(entries_)
+    print(serialized_entries_)
+    return {
+        "result": serialized_entries_,
+    }
+
+
+def _get_pvt_key():
+    secret_name = "publisherPrivateKey"
+    region_name = "us-west-1"
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
+    client = session.client(service_name="secretsmanager", region_name=region_name)
+    get_secret_value_response = client.get_secret_value(SecretId=secret_name)
+    return int(
+        json.loads(get_secret_value_response["SecretString"])["PUBLISHER_PRIVATE_KEY"]
+    )
+
+
+async def _handler(assets):
+    publisher = os.environ.get("PUBLISHER")
```

Comment:
Could use the `publish_all` function from stagecoach/jobs/publishers/examples/publish_all.py to avoid duplication?

Reply:
probably better to separate these since this uses the boto3 library to access secrets manager